packages/material-ui/src/Collapse/Collapse.test.js

```diff
@@ -14,15 +21,23 @@ describe('<Collapse />', () => {
   };
 
   before(() => {
+    mount = createMount();
     shallow = createShallow({ dive: true });
     classes = getClasses(<Collapse {...props} />);
   });
 
-  it('should render a Transition', () => {
-    const wrapper = shallow(<Collapse {...props} />);
-    assert.strictEqual(wrapper.name(), 'Transition');
+  after(() => {
+    mount.cleanUp();
   });
 
+  describeConformance(<Collapse {...props} />, () => ({
+    classes,
+    inheritComponent: 'Transition',
+    mount,
+    skip: ['refForwarding'],
+    testComponentPropWith: false,
```

Comment:
Why not using `skip`?

Reply:
This will be removed in a final cleanup. This is legacy from before `skip` was available and when I had plans to test that it would not accept a `component` prop. It's consistent with the rest of the codebase.